deploy/determined_deploy/aws/templates/govcloud.yaml

```diff
@@ -0,0 +1,708 @@
+---
```

Comment:
Meta-comment: I was gonna say "boy this looks like a lot of code that isn't Gov-specific, but I see we already have that for simple.yaml vs secure.yaml, etc. I guess there's not good solution to modularizing or inheritance beween templates...

Reply:
Agreed I'm sure there's a more efficient way to do this but thought I'd start by following our existing convention.